src/plib/library/EventListener.php

```diff
@@ -13,11 +13,11 @@ public function filterActions()
 
     public function handleEvent($objectType, $objectId, $action, $oldValues, $newValues)
     {
-        switch ($action) {
-            case 'domain_create':
-            case 'site_create':
-                $domainConnect = new \PleskExt\DomainConnect\DomainConnect(\pm_Domain::getByDomainId($objectId));
-                $domainConnect->enable();
+        if (in_array($action, ['domain_create', 'site_create'])) {
```

Comment:
Please, extend the list of events with 'site_create' and 'site_subdomain_create'